### ISSUE TITLE
'ascii' codec can't decode 

### ISSUE THREAD

**Issue by linlihai** [sripathikrishnan/redis-rdb-tools#66]
Use command 
`rdb --command protocol ~/Downloads/842a7322da7611e4-2016-04-06-11-28-00.rdb`

```
Traceback (most recent call last):
  File "/usr/local/bin/rdb", line 9, in <module>
    load_entry_point('rdbtools==0.1.6', 'console_scripts', 'rdb')()
  File "/Library/Python/2.7/site-packages/rdbtools/cli/rdb.py", line 82, in main
    parser.parse(dump_file)
  File "/Library/Python/2.7/site-packages/rdbtools/parser.py", line 306, in parse
    self.read_object(f, data_type)
  File "/Library/Python/2.7/site-packages/rdbtools/parser.py", line 369, in read_object
    self._callback.rpush(self._key, val)
  File "/Library/Python/2.7/site-packages/rdbtools/callbacks.py", line 340, in rpush
    self.emit('RPUSH', key, value)
  File "/Library/Python/2.7/site-packages/rdbtools/callbacks.py", line 298, in emit
    self._out.write(u"$" + unicode(len(unicode(arg))) + u"\r\n")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 9: ordinal not in range(128)
```

**Comment by chrisferry** [comment]
Shouldn't it be using UTF-8?  I'm having the same exact issue.

**Comment by mabdullah353** [comment]
I am stuck on this issue also, any solution for it ?

**Comment by oranagra** [comment]
fixed by #77
